ams/server/src/main/java/com/netease/arctic/server/utils/PreconditionUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.netease.arctic.server.utils;
+
+import com.netease.arctic.server.exception.AlreadyExistsException;
+import com.netease.arctic.server.exception.ObjectNotExistsException;
+
+public class PreconditionUtils {
+
+  public static void checkExist(boolean exists, String objectName) {
+    if (!exists) {
+      throw new ObjectNotExistsException(objectName);
+    }
+  }
+
+  public static void checkNotExist(boolean exists, String objectName) {
+    if (exists) {
+      throw new AlreadyExistsException(objectName);
+    }
+  }
```

Comment:
Could we use guava Preconditions instead ?

Reply:
guava is not suitable here